### PR TITLE
Restyle data source info in AppBar

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**
None (behind feature-flag)

**Description**
Update styling on data source in topnav.
- Remove border from initial "Open data source" button.
- Remove the down arrow

### CODE REVIEW

**Comment by 2metres** [comment]
Need help getting the Layout name. Currently it's fairly difficult

**Comment by jtbandes** [comment]
Can we make the layout name also be present when no data source is selected?

**Comment by 2metres** [comment]
@foxymiles I've managed to retrieve the Layout ID but could you help to return the layout name?

**Comment by amacneil** [comment]
- move layouts to the right
- make the text white
- keep the data source clickable for now (as you have in this pr)
- probably drop the external link icon for now

**Review by foxymiles — APPROVED** [review]

File: packages/studio-base/src/components/AppBar/DataSource.tsx [hunk 1]
```diff
@@ -2,14 +2,16 @@
 // License, v2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
-import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
-import ErrorIcon from "@mui/icons-material/Error";
-import { Button, ButtonBase, CircularProgress, Divider, Tooltip, Typography } from "@mui/material";
+import { ErrorCircle20Filled, Open16Filled } from "@fluentui/react-icons";
```
Comment (defunctzombie) [inline packages/studio-base/src/components/AppBar/DataSource.tsx:5]: What relationship does this change (styling the data source) have to the PR? If we can we should leave unrelated changes for separate PRs so when we review we know what change we are reviewing.

**Review by defunctzombie — CHANGES_REQUESTED** [review]
Putting a hold on this PR from slack discussion. We'll use a different approach here.

**Comment by defunctzombie** [comment]
I think we should replace this PR with https://github.com/foxglove/studio/pull/5687 and move the data source changes to a separate PR.

File: packages/studio-base/src/components/AppBar/DataSource.tsx (hunk 1 above)
Comment (amacneil) [reply packages/studio-base/src/components/AppBar/DataSource.tsx:5]: That was actually the original project. It morphed into mostly a layout menu change and reverting most (I thought all but I guess not) of the data source name changes.

image

**Review by defunctzombie — APPROVED** [review]

**Pull request merged**